docs/source/sctool/task.rst

```diff
@@ -71,6 +65,8 @@ You will have to create a new one.
    sctool task delete -c prod-cluster repair/143d160f-e53c-4890-a9e7-149561376cfd
 
 
+.. _task-history:
+
 task history
```

Comment:
@mmatczuk I couldn't find an equivalent command for ``task history``. Should we delete it from the docs (rst)?